lib/gs_schemas/capnp/game_types.capnp

```diff
@@ -102,3 +102,21 @@ struct FullChunkData {
     blockPalette @0 :List(UInt64);
     blockData @1 :List(UInt16);
 }
+
+# Action Data
+struct PositionData {
+    position @0 :IVec3;
+    offset @1 :Vec3;
+    look @2 :Vec3;
+}
+
+struct ThrowAction {
```

Comment:
The names here don't make much sense and I don't think this struct/enum is needed at all